pages/common/tldr.md

```diff
@@ -3,15 +3,15 @@
 > Displays simple help pages for command-line tools, from the tldr-pages project.
 > More information: <https://tldr.sh>.
 
-- Get typical usages of a command (hint: this is how you got here!):
+- Show the tldr page for a command (hint: this is how you got here!):
 
 `tldr {{command}}`
 
-- Show the tar tldr page for Linux:
+- Show the tldr page for `cd`, overriding the default platform:
 
-`tldr -p {{linux}} {{tar}}`
+`tldr -o {{windows}} {{cd}}`
```

Comment:
```suggestion
`tldr -p {{windows}} {{cd}}`
```
https://github.com/tldr-pages/tldr-node-client/pull/337

Reply:
There was a reason, we don't use `--platform` and `--update` here, right?

Reply:
Yep: https://github.com/tldr-pages/tldr/pull/5508#issuecomment-808312905

Reply:
Ah, I knew there was something.